proto/minder/v1/minder.proto

```diff
@@ -2951,7 +3035,7 @@ message ListEvaluationHistoryRequest {
         (buf.validate.field).repeated = {
             items: {
                 string: {
-                    pattern: "[,[:word:]]*",
+                    pattern: "^[,[:word:]]*$",
```

Comment:
Again, `,` seems like an error converting to a repeated field

Reply:
Because of - https://github.com/mindersec/minder/blob/3872633d436f63fda7c3f0cb842258153946f6bb/internal/controlplane/handlers_evalstatus.go#L283-L287